tinyslam_run.launch

```diff
@@ -14,9 +14,20 @@
     <param name="occupancy_estimator" value="const" />
     <param name="skip_exceeding_lsr_vals" value="false" />
 
+    <!-- Map setup-->
+    <param name="map_height_in_meters" value="50" />
+    <param name="map_width_in_meters" value="50" />
+    <param name="map_meters_per_cell" value="0.1" />
+
     <!-- Internal constants setup -->
     <param name="base_occupied_prob" value="0.95" />
     <param name="base_empty_prob" value="0.01" />
+    <param name="sigma_XY_MonteCarlo" value="0.2" />
```

Comment:
create either the "scan mather" comment to params or a prefix to the param name.